xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
```

Comment:
Hmm, [this doesn't look too bad](https://docs.mongodb.com/manual/core/document/#dot-notation) if `db()` supports it.